Autopilot/Path Manager/I2C.h.orig

```diff
@@ -0,0 +1,28 @@
+/* 
+ * File:   I2C.h
+ * Author: Chris Hajduk
+ *
+ * Created on March 10, 2013, 7:58 PM
```

Comment:
This file should not be versioned controlled. It is a merge conflict resolution intermediate file.